css/style.css

```
@@ -2,17 +2,15 @@
 
 @font-face {
   font-family: "Roboto";
-  src: local("Roboto"),
-    url(../fonts/roboto.woff2) format("woff2"),
-    url(../fonts/roboto.woff) format("woff ");
+  src: url(../fonts/roboto.woff2) format("woff2"),
+    url(../fonts/roboto.woff) format("woff");
```

Я кстати для красоты так пишу. И свойства одно под другим читать удобно, и отступы сохраняются
```suggestion
  src:
    url(../fonts/roboto.woff2) format("woff2"),
    url(../fonts/roboto.woff) format("woff");
```